Userland/Libraries/LibWeb/XHR/XMLHttpRequest.cpp

```diff
@@ -342,9 +352,8 @@ static Fetch::BodyWithType extract_body(XMLHttpRequestBodyInit const& body)
 
     // FIXME: 7. If source is a byte sequence, then set action to a step that returns source and length to source’s length.
     // FIXME: 8. If action is non-null, then run these steps in in parallel:
-    // NOTE: Step 9 is handled in the sub-steps of step 6 above.
-    // 9. Let body be a body whose stream is stream, source is source, and length is length.
```

Comment:
Ah I see now, unfortunately this looks like a rebase mistake - comment should be updated in pevious commit :)